src/main/java/uk/gov/hmcts/darts/common/repository/ExternalObjectDirectoryRepository.java

```diff
@@ -305,4 +305,14 @@ boolean hasMediaNotBeenCopiedFromInboundStorage(MediaEntity media, ObjectRecordS
                                                     ExternalLocationTypeEntity inboundLocation,
                                                     ObjectRecordStatusEntity ignoredUnstructuredStatus,
                                                     List<ExternalLocationTypeEntity> destinationLocations);
+
+    @Query(
+        """
+            SELECT eod FROM ExternalObjectDirectoryEntity eod
+            WHERE eod.status = :status and
```

Comment:
minor gripe - since "and" is sql keyword - should it be upper case? (and on next line?)